sfge/src/main/java/com/salesforce/graph/vertex/ModifierNodeVertex.java

```diff
@@ -67,6 +67,10 @@ public boolean isGlobal() {
         return getBoolean(Schema.GLOBAL);
     }
 
+    public boolean isImplicit() {
```

Comment:
`IS_IMPLICIT` is a property on the Method vertex (see `MethodWrapper.java`), not the ModifierNode vertex. The ModifierNode represents keywords like "global", "static", etc.
So this method belongs on the `MethodVertex`.

Reply:
Ah thanks. Yeah still learning the code here.

Reply:
Fixed (rebased and force pushed to update).